webpack.common.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const _ = require('underscore');
+const pkg = require('./package.json');
+
+module.exports = {
+    mode: 'production',
+    entry: './index.js',
+    resolve: {
+        extensions: ['.jsx', '.js'],
+    },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /(node_modules)/,
+                use: {
+                    loader: 'babel-loader',
+                },
+            },
+        ],
+    },
+    externals: [
+        'react-native',
+        /^expensify-common\/.+$/,
+        /^lodash\/.+$/,
+        ..._.keys(pkg.peerDependencies),
+        ..._.keys(pkg.dependencies),
```

Comment:
Our package `dependencies` are installed automatically into the consumer project `node_modules` 

`peerDependencies` would be installed as well, or flagged as not met depending on the `npm` version 
These are relatively easy to resolve if not installed, because we'll get an error that a module cannot be found